### ISSUE TITLE
Support Schemes Having Multiple Voting Machines

### ISSUE THREAD

**Issue by dOrgJelli** [dOrgTech/DAOcreator#188]
The SchemeRegistrar can have 2 voting machine configurations. This is partially supported in the library, and not supported at all in the UI.

**Comment by orishim** [comment]
I don't think we should work on supporting functionality in the DAOcreator that is not supported in Alchemy (for now)

**Comment by dOrgJelli** [comment]
No I'm referring to the Scheme Configuration UI in the DAOcreator. I think Alchemy actually supports this by default, not sure.
